tests/framework/common.go

```diff
@@ -1386,5 +1387,32 @@ func (td *OsmTestData) GrabLogs() error {
 		}
 	}
 
+	// TODO: Migrate these config collections to CLI
```

Comment:
What does this comment imply? Which config?

Reply:
the "config collection"

Reply:
I think you probably meant getting all configs with a single command.

Reply:
```suggestion
	// TODO: Move the logic to CLI so logic is reusable 
```

Reply:
@shashankram better?

Reply:
```suggestion
	// TODO: Eventually a CLI command should implement collection of configurations necessary for debugging
```